docker/android-slave/Dockerfile

```diff
@@ -16,13 +16,62 @@ RUN yum install -y centos-release-scl-rh && \
     INSTALL_PKGS="rh-nodejs4 rh-nodejs4-npm rh-nodejs4-nodejs-nodemon" && \
     ln -s /usr/lib/node_modules/nodemon/bin/nodemon.js /usr/bin/nodemon && \
     yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
+    rpm -V $INSTALL_PKGS
+
+# Install Java
+RUN INSTALL_PKGS="java-1.8.0-openjdk-devel" && \
+    yum install -y --enablerepo=centosplus $INSTALL_PKGS && \
     rpm -V $INSTALL_PKGS && \
     yum clean all -y
 
 RUN chown -R 1001:0 $HOME && \
     chmod -R g+rw $HOME
 
-## TODO Extend this to include android build tools
-## See https://github.com/appunite/docker/tree/master/android-java8-r25 
+# Install build dependencies
+RUN yum install -y git wget python curl 
+## Support native builds
+# RUN yum install gcc gcc-c++ make openssl-devel
```

Comment:
We probably need to install native tooling and support native builds. Maybe we can create separate Jenkinsfile for native stuff? Create some low priority jira for that?